tests/integration/test_parallel_replicas_no_replicas/test.py

```diff
@@ -48,3 +48,20 @@ def test_skip_all_replicas(start_cluster, skip_unavailable_shards):
                 "skip_unavailable_shards": skip_unavailable_shards,
             },
         )
+
+@pytest.mark.parametrize("max_parallel_replicas", [2, 3, 100])
```

Comment:
Please add the parameter to the existing test above. There is no need to create seconds test.